Automattic/wp-calypso
# Jetpack Products: Update the new constants for realtime products

#### Changes proposed in this Pull Request

Adding the new realtime products to the Store caused some of the product slugs to change from their original designs, so this PR updates the product slugs that were added in https://github.com/Automattic/wp-calypso/pull/54222/ to reflect that.

#### Testing instructions



1. Verify that all of the constants have been updated appropriately.
2. Verify that the pricing page has not changed.



Related to #
